R/scripts/create_filter.R

```diff
@@ -5,12 +5,15 @@ library(tidyr)
 library(purrr)
 
 option_list = list(
-  optparse::make_option(c("-c", "--config"), action="store", default='config.yml', type='character', help="path to the config file")
+  optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("CONFIG_PATH"), type='character', help="path to the config file")
```

Comment:
isn't this just going to lead to people setting their CONFIG_PATH to config.yml?

Reply:
i think most folks would either set the `CONFIG_PATH` environment variable once and forget about it or use the `-c` command line flag. implicitly loading a config file based on the directory in which you executed the script can cause issues (it loaded the wrong config file when i did the last runs).

`CONFIG_PATH` or `-c` will be a fully-qualified filename (e.g., '~/config.yml`). i guess you could just set it to "config.yml" naked but then you *explicitly* know you want to load the `config.yml` that exists in whatever directory you're currently running in. 

when running on a cluster it's better to have this global variable in the environment rather than being passed via command line args to every part of the chain.